packages/model-viewer/src/template.ts

```diff
@@ -344,6 +340,17 @@ canvas.show {
 
   <div class="slot default">
     <slot></slot>
+    
+    <div class="slot exit-webxr-button">
```

Comment:
Just my two cents, but I would give this a name that describes its semantic purpose. WebXR is a big umbrella, and doesn't necessarily capture what the user is trying to do when configuring the slot (it just adds ambiguity to the purpose of the button).

Reply:
Can you be more specific (or recommend a name)? `exit-webxr-button` seems pretty precise to me, but I'm probably just looking at it differently.

Reply:
I think this button is specifically used for exiting AR. If you were to support VR some day, you might do it via WebXR, but I suspect that this slot wouldn't apply (or at least should be interpreted differently). Maybe I'm wrong.

Just speaking generally, WebXR is a body of API around which many, various UX can be realized (not necessarily AR-specific UX).